src/js2xbox/events.cpp

```diff
@@ -236,4 +242,4 @@ void events::print_info_js(const std::string& fname) {
 			}
 		}
 	}
-}
\ No newline at end of file
+}
```

Comment:
Sorry just woke up, but here from line 81  to 239 there are no changes right?

Reply:
No, just my editor removing trailing whitespaces.